src/Layers/FeatureLayer/FeatureManager.js

```diff
@@ -383,22 +383,41 @@
       return this._service.query();
     },
 
+    _getMetadata: function(callback, context){
+      if(this._metadata){
+        callback(undefined, this._metadata, this);
```

Comment:
This should be `callback.call(context, error, this._metadata)` don't pass `context` to the callback function. Using `call` with context as the first param mean that `this` inside the callback function will equal `context`.